crypto/rsa/rsa_sign.c

```diff
@@ -97,6 +107,16 @@ static const unsigned char digestinfo_mdc2_der[] = {
       ASN1_OCTET_STRING, MDC2_DIGEST_LENGTH
 };
 # endif
+# ifndef OPENSSL_NO_RIPEMD160
```

Comment:
legacyprov.c uses OPENSSL_NO_RMD160
Replace any references to OPENSSL_NO_RIPEMD160

Reply:
Fixed